app/src/main/AndroidManifest.xml

```diff
@@ -32,7 +32,8 @@
         android:theme="@style/AppTheme"
         android:usesCleartextTraffic="true"
         tools:ignore="GoogleAppIndexingWarning"
-        tools:targetApi="m">
+        tools:targetApi="m"
```

Comment:
Will this break support for older Android versions than 6.0?

Reply:
"Indicates that Lint should treat this type as targeting a given API level, no matter what the project target is"
=> No affect anywhere. I should by the way delete this completely, it's some old stuff.

Reply:
https://github.com/norkator/paketin-seuranta/pull/35/commits/bc8b05010d741deee5de89325eb30ece76c92e23
Deleted it.

Reply:
@ristomatti one other note.. this app has not supported < Android 6.0 (API level 23) for long time.

minSdkVersion 23
targetSdkVersion 29